### PR TITLE
Updated log entries for various factions

### PR DESCRIPTION
**Pull request by pega3**
**Content (Artwork / Missions / Jobs)**

## Summary
This addresses issue #5124 

The PR adds log entries for the following factions:
- Unfettered Hai: after they have started to invade Wanderer territory and the Pug posted 2 warships
- Wanderers: after the Eye has opened and the Wanderers have started to establish colonies on the devastated Korath worlds
- Korath: after the Wanderers were able to end the Automata war
- Free Worlds: after the main plot is finished, providing different entries depending if Reconciliation or Check Mate was chosen
- Syndicate: after the main plot is finished and the player chose the Reconciliation option

### CODE REVIEW

File: data/wanderer/wanderers middle.txt [hunk 1]
```diff
@@ -3392,6 +3392,8 @@ mission "Wanderers: Sestor: Final"
 		dialog phrase "generic waypoint on visit"
 	on complete
 		log "Discovered that when the Korath exiles shut down the Kor Sestor factory, they also took enough technology from the factory to be able to begin producing war drones of their own. It is possible that the exiles have just become a whole lot more dangerous than they used to be."
+		log "Factions" "Wanderers" `After the Eye opened, creating a wormhole link between Wanderer territory and the region of space that harbours the devestated Korath worlds, the Wanderer have established several new colonies in the Korath sector and have begun to migrate to this area of Space. They have established formal relations with the Kor Efreti and were able to end the war between the Kor Mereti and the Kor Sestor.`
```
Comment (jvpoz) [inline data/wanderer/wanderers middle.txt:3395]:
```suggestion
		log "Factions" "Wanderers" `After the Eye opened, creating a wormhole link between Wanderer territory and the region of space that harbours the devestated Korath worlds, the Wanderers have established several new colonies in the Korath sector and have begun to migrate to this area of Space. They have established formal relations with the Kor Efreti and were able to end the war between the Kor Mereti and the Kor Sestor.`
```
typo

File: data/wanderer/wanderers start.txt [hunk 2]
```diff
@@ -2612,6 +2612,7 @@ mission "Wanderers Ap'arak 3"
 		has "Wanderers Ap'arak 2: done"
 	on offer
 		log `The Wanderers have begun producing even larger warships, and have succeeded in driving off yet another Unfettered assault. Afterwards the Pug, whom the Wanderers refer to as the "Keepers," finally sent assistance... in the form of two ships, which they claim are powerful enough to keep the entire Unfettered fleet at bay.`
+		log "Factions" "Unfettered Hai" `After obtaining a sufficient number of Jump Drives, the Unfettered expanded into Wanderer space and managed to conquer 3 worlds from the Wanderers until their advance was stopped by the interference of the Pug.`
```
Comment (jvpoz) [inline data/wanderer/wanderers start.txt:2615]:
```suggestion
		log "Factions" "Unfettered Hai" `After obtaining a sufficient number of Jump Drives, the Unfettered expanded into Wanderer space and managed to conquer 3 worlds from the Wanderers before their advance was stopped due to Pug interference.`
``` 
until --> before | I think it sounds better
Also shortened up the very end of the sentence.

**Comment by pega3** [comment]
@TideSinger @jvpoz @SubstandardZeal  - thank you for the reviews / approvals, I have committed all suggestions.

File: data/human/free worlds checkmate.txt [hunk 3]
```diff
@@ -2347,6 +2347,7 @@ mission "FWC End"
 	on offer
 		log "Parliament has agreed to a peace treaty, because the threat of alien invasion now seems much more pressing than the civil war with the Free Worlds. Whatever the true intention of the Pug was, the result of their interference is that humanity is at peace once more. There has been great loss of life, but nothing compared to what would have happened if the Free Worlds had tried to conquer Earth."
 		log "It may be a while before members of the Free Worlds manage to re-earn the trust of the rest of humanity, however."
+		log "Factions" "Free Worlds" `Tensions between the Free Worlds and the Republic mounted when additional systems decided to leave the Republic and join the Free Worlds, culminating in a series of military confrontations, that ended when the Pug invaded human space. After humanity united to drive out the Pug invaders, the Republic officially granted the Free Worlds autonomy.`
```
Comment (Amazinite) [inline data/human/free worlds checkmate.txt:2350]: This comma seems extraneous to me.
```suggestion
		log "Factions" "Free Worlds" `Tensions between the Free Worlds and the Republic mounted when additional systems decided to leave the Republic and join the Free Worlds culminating in a series of military confrontations, that ended when the Pug invaded human space. After humanity united to drive out the Pug invaders, the Republic officially granted the Free Worlds autonomy.`
```

File: data/human/free worlds reconciliation.txt [hunk 4]
```diff
@@ -2540,6 +2540,8 @@ mission "FW Syndicate Extremists 1C"
 		dialog `You land on <planet>, but realize that your escort carrying Freya hasn't entered the system yet. Better depart and wait for it to arrive.`
 	on complete
 		log "Received an official commendation from Parliament and a promise that the war with the Free Worlds is officially over. Humanity is at peace once more, but new challenges may await in the star systems out beyond human space that are accessible only by jump drive."
+		log "Factions" "Free Worlds" `Tensions between the Free Worlds and the Republic mounted when additional systems decided to leave the Republic and join the Free Worlds, culminating in a series of military confrontations, that ended when the Free Worlds were able to present credible evidence denying any involvement in the terrorist attacks. After the Free Worlds joined the effort to drive out the Pug invaders, the Republic officially recognized the Free Worlds and diplomatic relations were established.`
```
Comment (Amazinite) [inline data/human/free worlds reconciliation.txt:2543]:
```suggestion
		log "Factions" "Free Worlds" `Tensions between the Free Worlds and the Republic mounted when additional systems decided to leave the Republic and join the Free Worlds, culminating in a series of military confrontations that ended when the Free Worlds were able to present credible evidence denying any involvement in the terrorist attacks. After the Free Worlds joined the effort to drive out the Pug invaders, the Republic officially recognized the Free Worlds and diplomatic relations were established.`
```

File: data/human/free worlds reconciliation.txt [hunk 5]
```diff
@@ -2540,6 +2540,8 @@ mission "FW Syndicate Extremists 1C"
...
+		log "Factions" "Syndicate" `The reputation of the Syndicate took further damage when it became apparent that a group of top management executives had conspired with the Alphas in an attempt to create a war between the Republic and the Free Worlds.`
```
Comment (Amazinite) [inline data/human/free worlds reconciliation.txt:2544]: The way this log is worded sounds to me like it's coming after something else; "took further damage" implies to me that there's a prior sentence talking about an initial point in which their reputation was damaged, but this log doesn't exactly flow to me with the logs from the beginning of the game. (Even if it did flow, this wording then wouldn't make sense as soon as any other logs were added prior to this one.)

> in an attempt

They succeeded though, so I wouldn't simply call it an attempt.

How about this?
```suggestion
		log "Factions" "Syndicate" `It was exposed that the bombings of Gemini and Martini that helped cause the war between the Republic and Free Worlds were the doing of the Syndicate, designed by top executives with the help of a group of Alphas. Despite the claims that all the extremists responsible have been purged and handed over to the Navy, the reputation of the Syndicate will be damaged for years to come.`
```

File: data/wanderer/wanderers middle.txt [hunk 6]
```diff
@@ -3392,6 +3392,8 @@ mission "Wanderers: Sestor: Final"
 		dialog phrase "generic waypoint on visit"
 	on complete
 		log "Discovered that when the Korath exiles shut down the Kor Sestor factory, they also took enough technology from the factory to be able to begin producing war drones of their own. It is possible that the exiles have just become a whole lot more dangerous than they used to be."
+		log "Factions" "Wanderers" `After the Eye opened, creating a wormhole link between Wanderer territory and the region of space that harbours the devestated Korath worlds, the Wanderers have established several new colonies in the Korath sector and have begun to migrate to this area of Space. They have established formal relations with the Kor Efreti and were able to end the war between the Kor Mereti and the Kor Sestor.`
```
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3395]:
```suggestion
		log "Factions" "Wanderers" `After the Eye opened, creating a wormhole link between Wanderer territory and the region of space that harbors the devastated Korath worlds, the Wanderers have established several new colonies in the Korath sector and have begun to migrate to this area of Space. They have established formal relations with the Kor Efreti and were able to end the war between the Kor Mereti and the Kor Sestor.`
```

File: data/wanderer/wanderers middle.txt [hunk 7]
```diff
@@ -3392,6 +3392,8 @@ mission "Wanderers: Sestor: Final"
...
+		log "Factions" "Korath" `The Wanderers, who have begun to establish colonies on some deserted Korath systems, were able to end the war between the Kor Mereti and the Kor Sestor. The Kor Sestor have begun to rebuild their worlds and are tutored in this endeavor by the Wanderers.`
```
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3396]:
```suggestion
		log "Factions" "Korath" `The Wanderers, who have begun to establish colonies on some deserted Korath systems, were able to end the war between the Kor Mereti and the Kor Sestor. The Kor Efreti have begun to rebuild their worlds and are tutored in this endeavor by the Wanderers.`
```

File: data/wanderer/wanderers start.txt [hunk 8]
```diff
@@ -2612,6 +2612,7 @@ mission "Wanderers Ap'arak 3"
 		has "Wanderers Ap'arak 2: done"
 	on offer
 		log `The Wanderers have begun producing even larger warships, and have succeeded in driving off yet another Unfettered assault. Afterwards the Pug, whom the Wanderers refer to as the "Keepers," finally sent assistance... in the form of two ships, which they claim are powerful enough to keep the entire Unfettered fleet at bay.`
+		log "Factions" "Unfettered Hai" `After obtaining a sufficient number of Jump Drives, the Unfettered expanded into Wanderer space and managed to conquer 3 worlds from the Wanderers before their advance was stopped due to Pug interference.`
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:2615]: The Unfettered actually occupy six worlds before being stopped.
```suggestion
		log "Factions" "Unfettered Hai" `After obtaining a sufficient number of Jump Drives, the Unfettered expanded into Wanderer space and managed to conquer six worlds from the Wanderers before their advance was stopped due to Pug interference.`
```

File: data/wanderer/wanderers middle.txt (hunk 7 above)
Comment (Corraban2) [reply data/wanderer/wanderers middle.txt:3396]: Wouldn't this be the mereti?

File: data/human/free worlds reconciliation.txt (hunk 5 above)
Comment (pega3) [reply data/human/free worlds reconciliation.txt:2544]: Your version is better, so I committed it, however, the current entry ends with: Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.
That's the damaged reputation I was referring to.

**Comment by pega3** [comment]
@Amazinite - thanks for the review and improvements, I have committed everything. For some reason I must have missed the notification that you performed the review and only saw this now.

**Pull request merged**